packages/presentation/lib/screens/splash/data/splash_data.dart

```diff
@@ -0,0 +1,7 @@
+class SplashData {
```

Comment:
probably you don't need splash data at all at the moment

Reply:
I use SplashData for init bloc, bloc requires something to be inited.

Reply:
Fixed.